packages/core/src/preview.ts

```diff
@@ -9,7 +9,14 @@ import {
   prettyRawFunctionCall,
 } from './utils'
 
-type PreviewElement = DecodedAction | { to: string; data: string }
+type SystemDeploymentElement = {
```

Comment:
Added a new `PreviewElement` for deploying the system contracts. The system contracts are displayed as a single line in the preview